x-pack/plugins/security_solution/public/timelines/components/side_panel/event_details/index.tsx

```diff
@@ -150,41 +142,123 @@ const EventDetailsPanelComponent: React.FC<EventDetailsPanelProps> = ({
         onAddIsolationStatusClick={showHostIsolationPanel}
         timelineId={timelineId}
       />
-    </>
-  ) : (
+    );
+
+    if (isFlyoutView) {
+      bodyContent = (
```

Comment:
Consider eliminating the need for this this mutable `bodyContent` variable by separating the conditional logic for displaying the header from the conditional logic for displaying the body, per the details in this [suggestion](https://github.com/elastic/kibana/pull/139881/files?diff=split&w=1#r965371500).